classes-metodos-atributos/exercicio-04/src/main/java/MyProgram.java

```diff
@@ -0,0 +1,10 @@
+
+public class MyProgram{
```

Comment:
Como o exercício pede duas contagens, faça apenas um main e mova o outro código (for) para um main só.